README.md

```diff
@@ -27,21 +27,44 @@ plugins:[
     {
         use: 'gridsome-plugin-pwa',
         options: {
+            // Service Worker Options
+            disableServiceWorker: true,
+            serviceWorkerPath: 'service-worker.js',
+            cachedFileTypes: 'js,json,css,html,png,jpg,jpeg,svg,gif',
+
+            // Manifest Options (https://developer.mozilla.org/en-US/docs/Web/Manifest)
+            manifestPath: 'manifest.webmanifest',
```

Comment:
Revert this back to `manifest.json`.

Reply:
`manifest.webmanifest` is the official standard. I guess some might have trouble with serving it in their web server of choice, so done.